Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -16,9 +16,11 @@ public class BellerophonManager: NSObject {
     /// The default initializer.
     ///
     /// - Parameter window: The root window.
-    public init(window: UIWindow) {
+    /// - Parameter shouldPresentViewForForceUpdate: If enabled, force update would also present a view taking over the screen
+    public init(window: UIWindow, shouldPresentViewForForceUpdate: Bool = false) {
```

Comment:
it's kind of annoying that we have the append this to the init. It would be nice if the init just had parameters for the UIWindow. But I can't think of any alternatives off the top of my head, and you did mention to be backwards compatibility. If there was any alternatives I'd be open to them!

Reply:
I'd be open to just having a public ivar that is default to false so it does not change the init. But before I make that change, I'd like to know if any other developers have any opinions on this?

Reply:
Less so an opinion but more of an idea: How about asking the BPManagerDelegate whether or not force update should be handled?

```swift
@objc public protocol BellerophonManagerDelegate {
  ...
  func shouldManagerHandleForceUpdate(_ manager: BellerophonManager) -> Bool
  ...
}

// Defaulting force update handling to false for existing delegates, maintaining backwards compatibility
extension BellerophonManagerDelegate {
  func shouldManagerHandleForceUpdate(_ manager: BellerophonManager) -> Bool {
    return false
  }
}
```